phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -84,7 +84,7 @@
 
 			</div>
 		</div>
-
+<!-- EVENT overall_header_navbar_before -->
 		<!-- INCLUDE navbar_header.html -->
```

Comment:
But all events are added without any spaces in front of them, or are you saying to remove the spaces of INCLUDE?